packages/e2e-test-app/test/visitAllPages.test.ts

```diff
@@ -33,6 +33,9 @@ describe('visitAllPages', () => {
   }
 
   for (const api of apiExamples) {
+    if (api === 'Transforms')
+      // disable until either transformExample uses units, or that isn't an error
```

Comment:
Just curious on more context for this comment. Is this from Jest upgrade as well?

Reply:
This is a testpage that crashes in core as of this PR: https://github.com/facebook/react-native/pull/34660

https://github.com/facebook/react-native/pull/35292 will fix the test page